src/bindings/PyDP/algorithms/util.cpp

```diff
@@ -14,4 +15,29 @@ void init_algorithms_util(py::module& m) {
   util.def("default_epsilon", &dp::DefaultEpsilon);
   util.def("get_next_power_of_two", &dp::GetNextPowerOfTwo);
   util.def("qnorm", &dp::Qnorm);
-}
+  util.def("mean", &dp::Mean<double>);
+  util.def("mean", &dp::Mean<int>);
+  util.def("variance", &dp::Variance<double>);
+  util.def("standard_deviation", &dp::StandardDev<double>);
+  util.def("order_statistics", &dp::OrderStatistic<double>);
+  util.def("correlation", &dp::Correlation<double>);
+  util.def("vector_filter", &dp::VectorFilter<double>);
+  util.def("vector_to_string", &dp::VectorToString<double>);
+  util.def("round_to_nearest_multiple", &dp::RoundToNearestMultiple);
+  // TODO: Throw an exception in case the function returns false.
+  util.def("safe_add", [](int64_t i, int64_t j) {
```

Comment:
How to change datatype when we want to use floats?

Reply:
If we do something like `(double i, double j){}` it would automatically use that if passed a float.

Reply:
So the way we are supposed to use this is: 
`safe_add(number1, number2, 'int'/'float')`

Reply:
How would we do something like this in plain c++ ?